packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,53 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface UseFileDialogOptions {
+  multiple?: boolean
+  accept?: string
+}
```

Comment:
```suggestion
export interface UseFileDialogOptions {
  multiple?: boolean
  accept?: string
  /**
   * Select the input source for the capture file.
   * @see [HTMLInputElement Capture](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture)
   */
  capture?: string
}
```

Reply:
Hey sorry I didn't see this last commit suggestion. Anyway, I copy pasted it.